icu4c/source/test/cintltst/unumberrangeformattertst.c

```diff
@@ -59,6 +59,10 @@ static void TestExampleCode() {
     // Get the result string:
     int32_t len;
     const UChar* str = ufmtval_getString(unumrf_resultAsValue(uresult, &ec), &len, &ec);
+    if (str == NULL) {
```

Comment:
This bypasses the following assertSuccessCheck.

I think a better fix is probably to put a nullptr check into assertUEquals(). Something like
```
if (str == nullptr) {
    str = u"(null)";
}
```
For each of the expected/actual string pointers.

Reply:
I see that you changed *this* code to use that check. But as I said before, I think it would be better (= help other call sites) to do this *inside* assertUEquals(), for each of the two pointer arguments.

Also please make sure to use ICU indentation.